server/index.js

```diff
@@ -1,25 +1,26 @@
-const express = require('express');
+const express = require("express");
 const cors = require("cors");
-const wareHouseRouter = require('./routes/warehouses');
-const inventoryRouter = require('./routes/inventories');
+const wareHouseRouter = require("./routes/warehouses");
+const inventoryRouter = require("./routes/inventories");
 
-require('dotenv').config();
+require("dotenv").config();
 
 const app = express();
 
 app.use(express.json());
 
 const PORT = process.env.PORT || 8080;
 
-app.use(cors({
-  origin: process.env.CLIENT_URL
-}));
+app.use(
+  cors({
+    origin: process.env.CLIENT_URL,
```

Comment:
The code looks good. The only suggestion I would make is to erase the comma at the end of line 16.